collections/_docs/handbook/style/colons.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Colons
+key_point: A colon indicates that closely-related information follows.
+original_url: <https://developers.google.com/style/colons>
+original_title: Colons | Google developer documentation style guide
+---
+
+<section id="phrase-preceding-colon" markdown="1">
+
+## Introductory Phrase Preceding Colon
+
+When a colon introduces a list, the text that precedes the colon should
+ordinarily be able to stand alone as a complete sentence.
+
+<strong>Examples</strong>
+
+<p class="example">
+  <span class="compare-better">Recommended:</span> The fields are defined as
+  follows:
+</p>
+<p class="example">
+  <span class="compare-worse">Not recommended:</span> The fields are:
+</p>
+
+</section>
+
+<section id="usage-of-japanese-colon" markdown="1">
+
+## Proper Etiquette for Use of Japanese Colon
+
+We at OpenINF are always interested in finding ways to optimize things and be
+more efficient. One particular use-case of [the Japanese colon][] that we have
+picked-up is immediately after the classification portion of a commit message
+title.
+
+Rather than using an ordinary colon from the ANSI or Unicode character sets, we
+use the Japanese colon as it is both less size (in memory) and can accomplish
+the task of both a colon character and the subsequent space all at once.
+
+</section>
+
+<section id="bold-text-preceding-colon" markdown="1">
+
+## Bold and Italic Text Preceding Colons
+
+When non-italic (also known as _Roman_) text that precedes a colon is bold, the
+colon should not be made bold, italic, or otherwise (color-stylized, etc.).
```

Comment:
```suggestion
colon should not be made bold, italic, or otherwise (color-stylized, etc.).
```

is this contentious

Reply:
```suggestion
colon should not be made bold, italic, or otherwise (color-stylized, etc.).
This, however, is not a hard and fast rule. In general, use best intuition.
```